tests/conftest.py

```diff
@@ -16,3 +16,8 @@ def set_seed():
 @pytest.fixture(scope="session", autouse=True)
 def set_default_tensor_type():
     torch.set_default_tensor_type("torch.FloatTensor")
+
+
+@pytest.fixture(scope="function")
+def mcmc_params_testing() -> dict:
```

Comment:
Great, so these is the fixture fixing all global MCMC hyperparameters.

Are the current values, what is suggested as new default?

Reply:
great idea to add a fixture!

Reply:
These values are only used in the tests so far, and I would argue for keeping it this way. Here is why: we want to check if thinning causes errors (in all combinations), even if the new default will be 1, i.e., no thinning. We want the tests to run fast, hence the warmup is reduced to 50. There is no one good number for the burnin samples, but 100 is the least I found in other packages. However, for the tests, we usually don't care if the chain is already stationary (most of them are interface tests). The number of chains is debatable. I thought about using 1 but this rules out parallelism for the tests. 2 could obfuscate some shaping errors since thinning=2 (I know this is unlikely, but I am superstitious on these things). So I went with `num_chains=3`. For a real problem, a higher number of chains, e.g. 4 to 10, is typical I guess. 

Note that in the current state, sometimes the number of burin samples is 1 and the number of chains is 10, which is an extreme setting :boom: :rofl: